### DISCUSSION TITLE
Two accounts sharing the same pictures

### DISCUSSION THREAD

**Discussion by mungo312** [photoview/photoview#343] [Q&A and Troubleshooting] [answered]
Hi,

photoview is a realy great piece of software. I currently try to migrate from nextcloud to it for my pictures. I want to share the pictures with my wife. Should I create her an account in photoview with the same folder path as in my account ? Are the images then imported twice ? Or should I use the public share feature ? The account would be better because the link will get lost ... I know ...

**Comment by viktorstrate** [comment, marked as answer]
Hi @mungo312,

Thank you!
You can just make two accounts that share a common path, Photoview is smart enough to only process the media once but shares and favorites are tied to the individual users.
